### PR TITLE
Responsive support

### PR DESCRIPTION
**Pull request by Germain-Italic**
Attempts to fix responsive view by producing a 1-column albums list and a 2-columns photos list on iPhone 5 and larger.

Related issues : https://github.com/cshold/jQuery-Facebook-Photo-Selector/issues/25 and https://github.com/cshold/jQuery-Facebook-Photo-Selector/issues/16

Screenshots (project name blurred on purpose):

Albums view

Photos view

### CODE REVIEW

File: csphotoselector.css [hunk 1]
```diff
@@ -278,3 +278,41 @@
 .CSPhoto_container_active {
   left: 11px;
 }
+
+/* responsive support */
+/* optimized for iPhone 5 and upper
+ * will produce a 1-column albums list
+ * and a 2-columns photos list
+ */
+@media screen and (max-width: 480px) {
```
Comment (cshold) [inline csphotoselector.css]: Can you change this to 650px. Right now it gets cut off a bit between that and 480.

Comment (Germain-Italic) [inline csphotoselector.css]: Updated in https://github.com/cshold/jQuery-Facebook-Photo-Selector/pull/26/commits/9ad6795cd7f2676516f6a40328a92626fbb6d7df which also contains a fix for the Close button that was not correctly positioned (I have commited in the wrong branch the first time so I merged PhotosOfYou into Master)

Comment (cshold) [inline csphotoselector.css]: Ah k, should we close this PR then and just focus on that one?

Comment (Germain-Italic) [inline csphotoselector.css]: The close button commit is part of the mediaquery enhancements, so it's the same PR

File: index.html [hunk 2]
```diff
@@ -60,12 +60,11 @@
     <!-- Markup for Carson Shold's Photo Selector -->
     <div id="CSPhotoSelector">
       <div class="CSPhotoSelector_dialog">
-        <a href="#" id="CSPhotoSelector_buttonClose">x</a>
         <div class="CSPhotoSelector_form">
           <div class="CSPhotoSelector_header">
-            <p>Choose from Photos</p>
+              <a href="#" id="CSPhotoSelector_buttonClose">x</a>
+              <p>{{ 'facebook.photoselector.choose'|trans|raw }}</p>
```
Comment (cshold) [inline index.html]: What does this line do?

Comment (Germain-Italic) [inline index.html]: It proves that copy/pasting is not always a good idea.
Fixed in d49599e

**Comment by cshold** [comment]
👍  Going to merge this one in. Can't take a look at the other one until tomorrow likely but appreciate all the work on this.

**Comment by Germain-Italic** [comment]
Great!
I hope we get some feedbacks on the rendering, because I only have an iPhone SE / iPhone 6 / Galaxy S3 to test the mobile view without emulator.

**Pull request merged**